### DISCUSSION TITLE
how can I update osu Lazer Linux?

### DISCUSSION THREAD

**Discussion by cool271** [ppy/osu#29239] [Q&A]
also im on Chromebook so idk if it that's why appimage don't work

**Comment by ThunderGemios10** [comment]
The AppImage updater doesn't seem to work. If it's compatible for your device, you can try the Flatpak [here](https://flathub.org/apps/sh.ppy.osu).

**Reply by cool271** [reply]
after I download it and clicked it just show some coding. do you know the next step?

**Reply by Rudicito** [reply]
You need first to install Flatpak and add the Flathub repository : [link](https://flathub.org/setup/Chrome%20OS)

Try launching the file you downloaded earlier. if it show coding stuff again do that on a terminal:

`flatpak install flathub sh.ppy.osu`

**Comment by felipemarins** [comment]
AppImageUpdate works for me and I don't think there would be any difference if you're using linux on chromebook. I don't know about Chrome OS tho.
If you want to try it, you can download it here: https://github.com/AppImageCommunity/AppImageUpdate
After opening it, you have to select the osu appimage and then it will start updating.
